easybuild/easyblocks/i/impi.py

```diff
@@ -138,7 +138,7 @@ def post_install_step(self):
                     sys.stdout.write(line)
             for script in [os.path.join('intel64', 'bin', 'mpivars.sh'), os.path.join('mic', 'bin', 'mpivars.sh')]:
                 for line in fileinput.input(os.path.join(self.installdir, script), inplace=1, backup='.orig.eb'):
-                    line = re.sub(r"^I_MPI_ROOT=", "I_MPI_ROOT=%s; export I_MPI_ROOT" % self.installdir, line)
+                    line = re.sub(r"^I_MPI_ROOT=.*", "I_MPI_ROOT=%s; export I_MPI_ROOT" % self.installdir, line)
```

Comment:
Urgh, this should never have made it in like this, thanks @stachon!

How did we miss this in #629 @ocaisa?